app/views/providers/means_reports/_capital_result.html.erb

```diff
@@ -0,0 +1,29 @@
+
+<section class="print-no-break">
+  <h2 class="govuk-heading-m"><%= t(".heading") %></h2>
+
+  <%= govuk_summary_card(title: t(".heading"),
+                         html_attributes: { id: "app-check-your-answers__capital_result__card" }) do |card|
```

Comment:
No heading_level: 3 on this card, should there be?

Reply:
done